packages/app/src/components/Root/Root.tsx

```diff
@@ -249,6 +266,7 @@ export const Root = ({ children }: PropsWithChildren<{}>) => {
   };
   return (
     <SidebarPage>
+      <GlobalFABProvider context={fabContext} />
```

Comment:
I think you should also import that in AppBase, not Root.

The `ApplicationProvider` should wrap the <Root> there and the `ApplicationListener` should be after `</Root>` I guess.